CRM/Contribute/Form/Contribution/ThankYou.php

```diff
@@ -284,6 +283,28 @@ public function buildQuickForm() {
       $this->assign('friendURL', $url);
     }
 
+    $isPendingOutcome = TRUE;
+    try {
+      // A payment notification update could have come in at any time. Check at the last minute.
+      $contributionStatusID = civicrm_api3('Contribution', 'getvalue', array(
+        'id' => CRM_Utils_Array::value('contributionID', $params),
+        'return' => 'contribution_status_id',
+        'invoice_id' => CRM_Utils_Array::value('invoiceID', $params),
```

Comment:
yeah - the point is the ipn could have updated the status - so looking to find out what it actually is